osu.Game/Users/Drawables/StatusIcon.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osuTK;
+
+namespace osu.Game.Users.Drawables
+{
+    public partial class StatusIcon : CircularContainer
+    {
+        public StatusIcon()
+        {
+            Size = new Vector2(25);
+            BorderThickness = 4;
+            BorderColour = Colour;
```

Comment:
Big issue with this is that if you ever attempt to change `Colour` (as I figure you may want to, to show a different status as a different colour), then the change will not propagate back to `BorderColour`.

You probably want to `IHasAccentColour` this for proper operation.

Reply:
It will, actually. This is how status colour updates work already, updating `Colour` updates the `BorderColour` too

Reply:
Ok, on double-checking: I was wrong, but this is kinda obscure too, and it doesn't really work how you think it does. What this line *really* does, is it sets the border colour to the default initial colour of `Drawable`, which is white.

Now when you set `Colour` on a drawable, the colour is applied to the drawable *in a multiplicative manner*, i.e. the colour of all of the contents of the drawable are *multiplied* by `Colour`. White is a neutral colour to this multiplication, by which I mean that white multiplied by any colour is that same colour (because white is essentially all ones). This does *not* mean that this auto-transfers the value or something.

Taking all of the above into account I'd just set the colour to white to not mislead.

Reply:
Oh, that's indeed quite unintuitive - I'll change the colour to white and add a note about how it works